### ISSUE TITLE
[make failure] error: ‘__args_arr’, ‘__args_idx’ was not declared in this scope

### ISSUE THREAD

**Issue by oscarriddle** [onnx/onnx-tensorrt#150]
My OS info is :
CentOS7, cmake-3.14, tensorrt-5.0.2.6, gcc-4.9.3, protoc-3.7.0/3.7.1, CUDA9.0, cuDNN7.0.5

---------------------
The previous problem is solved by building protobuf from source, but then encountered a new problem, please refer to the next comment.

**Comment by oscarriddle** [comment]
I tried to git clone the master branch of protobuf and successfully build it.
Then cmake the onnx-tensorrt again and successfully done but with several warning:
```
-- Configuring done
WARNING: Target "getSupportedAPITest" requests linking to directory "/home/web_server/protobuf".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "onnx2trt" requests linking to directory "/home/web_server/protobuf".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "nvonnxparser" requests linking to directory "/home/web_server/protobuf".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "nvonnxparser_static" requests linking to directory "/home/web_server/protobuf".  Targets may link only to libraries.  CMake is dropping the item.
WARNING: Target "trt_onnxify" requests linking to directory "/home/web_server/protobuf".  Targets may link only to libraries.  CMake is dropping the item.
-- Generating done
-- Build files have been written to: /home/web_server/workshop/onnx-tensorrt/build
```
Then I tried to make and failed with errors as below:
```
$ make -j8
Scanning dependencies of target nvonnxparser_plugin
[  2%] Building CUDA object CMakeFiles/nvonnxparser_plugin.dir/FancyActivation.cu.o
[  5%] Building CUDA object CMakeFiles/nvonnxparser_plugin.dir/ResizeNearest.cu.o
[  7%] Building CUDA object CMakeFiles/nvonnxparser_plugin.dir/Split.cu.o
[ 12%] Building CXX object CMakeFiles/nvonnxparser_plugin.dir/InstanceNormalization.cpp.o
[ 12%] Building CXX object CMakeFiles/nvonnxparser_plugin.dir/plugin.cpp.o
In file included from /home/web_server/gcc-5.3/lib/gcc/x86_64-unknown-linux-gnu/5.3.0/include/stddef.h:217:0:
/usr/local/cuda/include/crt/host_runtime.h:19:2: warning: #warning "crt/device_functions.h is an internal header file and must not be used directly.  Please use cuda_runtime_api.h or cuda_runtime.h instead." [-Wcpp]
 #warning "crt/device_functions.h is an internal header file and must not be used directly.  Please use cuda_runtime_api.h or cuda_runtime.h instead."
  ^
In file included from /tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:5:0,
                 from tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:1:
/usr/local/cuda/include/crt/host_runtime.h:19:2: warning: #warning "crt/device_functions.h is an internal header file and must not be used directly.  Please use cuda_runtime_api.h or cuda_runtime.h instead." [-Wcpp]
 #warning "crt/device_functions.h is an internal header file and must not be used directly.  Please use cuda_runtime_api.h or cuda_runtime.h instead."
  ^
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c: In function ‘void __device_stub__Z24resize_nearest_kernel_2dIfEvi6float24int2PKT_iiPS2_ii(int, float2&, int2&, const float*, int, int, float*, int, int)’:
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:11:223: error: ‘__args_arr’ was not declared in this scope
 static void __device_stub__Z24resize_nearest_kernel_2dIfEvi6float24int2PKT_iiPS2_ii(int __par0, struct float2&__par1, struct int2&__par2, const float *__par3, int __par4, int __par5, float *__par6, int __par7, int __par8){__cudaSetupArgSimple(__par0, 0UL);__cudaSetupArg(__par1, 8UL);__cudaSetupArg(__par2, 16UL);__cudaSetupArgSimple(__par3, 24UL);__cudaSetupArgSimple(__par4, 32UL);__cudaSetupArgSimple(__par5, 36UL);__cudaSetupArgSimple(__par6, 40UL);__cudaSetupArgSimple(__par7, 48UL);__cudaSetupArgSimple(__par8, 52UL);__cudaLaunch(((char *)((void ( *)(int, struct float2, struct int2, const float *, int, int, float *, int, int))resize_nearest_kernel_2d<float> )));}
                                                                                                                                                                                                                               ^
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:11:223: error: ‘__args_idx’ was not declared in this scope
 static void __device_stub__Z24resize_nearest_kernel_2dIfEvi6float24int2PKT_iiPS2_ii(int __par0, struct float2&__par1, struct int2&__par2, const float *__par3, int __par4, int __par5, float *__par6, int __par7, int __par8){__cudaSetupArgSimple(__par0, 0UL);__cudaSetupArg(__par1, 8UL);__cudaSetupArg(__par2, 16UL);__cudaSetupArgSimple(__par3, 24UL);__cudaSetupArgSimple(__par4, 32UL);__cudaSetupArgSimple(__par5, 36UL);__cudaSetupArgSimple(__par6, 40UL);__cudaSetupArgSimple(__par7, 48UL);__cudaSetupArgSimple(__par8, 52UL);__cudaLaunch(((char *)((void ( *)(int, struct float2, struct int2, const float *, int, int, float *, int, int))resize_nearest_kernel_2d<float> )));}
                                                                                                                                                                                                                               ^
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c: In function ‘void __device_stub__Z24resize_nearest_kernel_2dI6__halfEvi6float24int2PKT_iiPS3_ii(int, float2&, int2&, const __half*, int, int, __half*, int, int)’:
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:13:245: error: ‘__args_arr’ was not declared in this scope
 static void __device_stub__Z24resize_nearest_kernel_2dI6__halfEvi6float24int2PKT_iiPS3_ii(int __par0, struct float2&__par1, struct int2&__par2, const struct __half *__par3, int __par4, int __par5, struct __half *__par6, int __par7, int __par8){__cudaSetupArgSimple(__par0, 0UL);__cudaSetupArg(__par1, 8UL);__cudaSetupArg(__par2, 16UL);__cudaSetupArgSimple(__par3, 24UL);__cudaSetupArgSimple(__par4, 32UL);__cudaSetupArgSimple(__par5, 36UL);__cudaSetupArgSimple(__par6, 40UL);__cudaSetupArgSimple(__par7, 48UL);__cudaSetupArgSimple(__par8, 52UL);__cudaLaunch(((char *)((void ( *)(int, struct float2, struct int2, const struct __half *, int, int, struct __half *, int, int))resize_nearest_kernel_2d< ::__half> )));}
                                                                                                                                                                                                                                                     ^
/tmp/tmpxft_00004640_00000000-5_ResizeNearest.compute_70.cudafe1.stub.c:13:245: error: ‘__args_idx’ was not declared in this scope
 static void __device_stub__Z24resize_nearest_kernel_2dI6__halfEvi6float24int2PKT_iiPS3_ii(int __par0, struct float2&__par1, struct int2&__par2, const struct __half *__par3, int __par4, int __par5, struct __half *__par6, int __par7, int __par8){__cudaSetupArgSimple(__par0, 0UL);__cudaSetupArg(__par1, 8UL);__cudaSetupArg(__par2, 16UL);__cudaSetupArgSimple(__par3, 24UL);__cudaSetupArgSimple(__par4, 32UL);__cudaSetupArgSimple(__par5, 36UL);__cudaSetupArgSimple(__par6, 40UL);__cudaSetupArgSimple(__par7, 48UL);__cudaSetupArgSimple(__par8, 52UL);__cudaLaunch(((char *)((void ( *)(int, struct float2, struct int2, const struct __half *, int, int, struct __half *, int, int))resize_nearest_kernel_2d< ::__half> )));}
```

**Comment by kevinch-nv** [comment]
Have you tried building with the latest TensorRT version (7.2)?
